### PR TITLE
created fbc_showToast

### PR DESCRIPTION
**Pull request by ericrsmith35**
Changed RecordDetail to use new fbc_showToast

### CODE REVIEW

File: flow_process_components/FlowBaseComponents/force-app/main/default/lwc/fbc_pickObjectAndField/fbc_pickObjectAndField.js-meta.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
     <apiVersion>46.0</apiVersion>
-    <description>Pick Object And Field FSC</description>
+    <description>fbc_PickObjectAndField</description>
```
Comment (alexed1) [inline flow_process_components/FlowBaseComponents/force-app/main/default/lwc/fbc_pickObjectAndField/fbc_pickObjectAndField.js-meta.xml:4]: -I0'd like to keep the descriptions and labels 'friendly'....  perhaps 'Pick Object and Field'[--m

File: flow_process_components/FlowBaseComponents/force-app/main/default/lwc/fbc_toastMessage/fbc_toastMessage.js-meta.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>46.0</apiVersion>
+    <description>fbc_toastMessage</description>
```
Comment (alexed1) [inline flow_process_components/FlowBaseComponents/force-app/main/default/lwc/fbc_toastMessage/fbc_toastMessage.js-meta.xml:4]: friendly names here too

File: flow_process_components/FlowBaseComponents/force-app/main/default/lwc/fbc_pickObjectAndField/fbc_pickObjectAndField.js-meta.xml (hunk 1 above)
Comment (ericrsmith35) [reply flow_process_components/FlowBaseComponents/force-app/main/default/lwc/fbc_pickObjectAndField/fbc_pickObjectAndField.js-meta.xml:4]: The other two that you exposed use the fbc_ label.  I was trying to be consistent.  Would you still like me to change this back?

Comment (alexed1) [reply flow_process_components/FlowBaseComponents/force-app/main/default/lwc/fbc_pickObjectAndField/fbc_pickObjectAndField.js-meta.xml:4]: Yes, when you showed me the screen shot I realized that I had failed to put good labels in. So change them here and there as well, if you don't mind

**Comment by ericrsmith35** [comment]
done

**Pull request closed without merging**